bundles/org.openhab.io.hueemulation/src/main/java/org/openhab/io/hueemulation/internal/upnp/UpnpServer.java

```diff
@@ -61,6 +61,7 @@
 import org.osgi.service.http.NamespaceException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import sun.nio.ch.Net;
```

Comment:
Hm. "sun" headers are generally deprecated I think. Is there an alternative in the JDK?

Reply:
No idea how it snuck in there. It was unused